wiki/Application-Issues.md

```diff
@@ -1,8 +1,27 @@
+### Electron based applications
+
+Electron based applications can work directly on Wayland, but they don't accept it without configuration. Configuration differs between versions. Information was grabbed from the [ArchWiki](https://wiki.archlinux.org/title/Wayland#Electron).
+
+For Electron (>28) you need to set environment variable:
+```kdl
+environment {
+    ELECTRON_OZONE_PLATFORM_HINT "auto"
+}
+```
+
+For previous version you need to set flags in configuration file `~/.config/electron-flags.conf`:
```

Comment:
This file is Arch-specific, so we can't document it here

Reply:
Oh, I thought it is electron based, but you are right. I will remove those and just paste about command line flags.

Reply:
Removed.